### PR TITLE
Get rid of the remains of heavy debug

### PR DESCRIPTION
**Pull request by bzsolt**
JerryScript-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
LGTM

**Comment by zherczeg** [comment]
LGTM

File: jerry-core/jrt/jrt-fatals.c [hunk 1]
```diff
@@ -82,15 +82,15 @@ jerry_assert_fail (const char *assertion, /**< assertion condition string */
                    const char *function, /**< function name */
                    const uint32_t line) /**< line */
 {
-#if !defined (JERRY_NDEBUG) || !defined (JERRY_DISABLE_HEAVY_DEBUG)
+#if !defined (JERRY_NDEBUG)
```
Comment (akiss77) [inline jerry-core/jrt/jrt-fatals.c]: Wouldn't this be nicer as `#ifndef JERRY_NDEBUG`?

File: jerry-core/jrt/jrt.h [hunk 2]
```diff
@@ -77,12 +77,12 @@ extern void __noreturn jerry_assert_fail (const char *, const char *, const char
 extern void __noreturn jerry_unreachable (const char *, const char *, const char *, const uint32_t);
 extern void __noreturn jerry_unimplemented (const char *, const char *, const char *, const uint32_t);
 
-#if !defined (JERRY_NDEBUG) || !defined (JERRY_DISABLE_HEAVY_DEBUG)
+#if !defined (JERRY_NDEBUG)
```
Comment (akiss77) [inline jerry-core/jrt/jrt.h]: same here

File: tests/unit/test-common.h [hunk 3]
```diff
@@ -26,11 +26,11 @@
 #include <string.h>
 
 /**
- * Verify that unit tests are built with all debug checks enabled
+ * Verify that unit tests are built with enabled debug check
  */
-#if defined (JERRY_NDEBUG) || defined (JERRY_DISABLE_HEAVY_DEBUG)
-# error "defined (JERRY_NDEBUG) || defined (JERRY_DISABLE_HEAVY_DEBUG) in a unit test"
-#endif /* JERRY_NDEBUG || JERRY_DISABLE_HEAVY_DEBUG */
+#if defined (JERRY_NDEBUG)
```
Comment (akiss77) [inline tests/unit/test-common.h]: And here it could be simply `#ifdef JERRY_NDEBUG`

**Comment by bzsolt** [comment]
@akiss77 Good point, I've updated the patch. Thanks!

**Comment by akiss77** [comment]
LGTM

**Pull request merged**